cmd/kube-proxy/app/server_others.go

```diff
@@ -55,6 +60,8 @@ import (
 	"k8s.io/klog"
 )
 
+var timeoutForNodePodCIDR = 5 * time.Minute
```

Comment:
comment?

Reply:
core idea here was to be resilient to failures in allocators assigning podCIDR. Since this is done asynchronously, we don't want to wait for ever for the podCIDR to be assigned and want to timeout and fail in case it does not happen. 5 minute was a safe timeout. Will add comment capturing above.

Reply:
Done.